scripts/stats-dashboard/mp-coverage.py

```diff
@@ -95,7 +95,7 @@ def main():
     baseline_df = pd.read_csv("corpus/quality_assessment/baseline_mp_frequencies_per_year/number_mp_in_parliament.csv")
```

Comment:
These should be provided as arguments–we are moving scripts to other locations soon anyway

Reply:
[7280c5b](https://github.com/welfare-state-analytics/riksdagen-corpus/pull/477/commits/7280c5b50508d9033a1252720825e30166986964)